src/app/pages/workspace/workspace-text-window/workspace-text-window.component.ts

```diff
@@ -107,7 +106,10 @@ export class WorkspaceTextWindowComponent implements OnInit, OnDestroy {
   /**Layer selezionato */
   selectedLayer: TLayer | undefined;
   /**Lista di layer selezionati */
-  selectedLayers: TLayer[] | undefined;
+  selectedLayers: TLayer[] | undefined = [];
+  startRow: number | null = null;
```

Comment:
come mai non usi this.textRange.start e this.textRange.end? Queste due proprietà mi sembrano una ripetizione

Reply:
Verificata la possibilità di utilizzare precTextRange, la sostituzione richiederebbe una modifica nel caricamento iniziale (alla prima esecuzione gli indici sono già valorizzati) e un controllo sui valori perché non contengano decimali, portando a preferire la ridondanza.